src/com/dabomstew/pkrandom/newgui/Bundle.properties

```diff
@@ -583,4 +583,19 @@ GUI.tmFollowEvolutionsCheckBox.toolTipText=<html>When this is selected and TM co
-GUI.stpFixMusicAllCheckBox.toolTipText=<html>Fixes the music for all Static Pokemon encounters so that even when randomized, encounters that should have special music will play the correct song.<br />Note that in Gen 4/5, if special music is assigned to a Pokemon then it will <b>always</b> play when battling that Pokemon, even if you just find it normally in the wild.
\ No newline at end of file
+GUI.stpFixMusicAllCheckBox.toolTipText=<html>Fixes the music for all Static Pokemon encounters so that even when randomized, encounters that should have special music will play the correct song.<br />Note that in Gen 4/5, if special music is assigned to a Pokemon then it will <b>always</b> play when battling that Pokemon, even if you just find it normally in the wild.
+GUI.heldItemsAdd=Add Held Items to...
+GUI.heldItemsRandom.text=Random
+GUI.heldItemsRandom.toolTipText=<html>Items will be chosen randomly from all held items that could have an effect on NPC pokemon.<br />This does not include items like Escape Rope (no held effect), Power Bracers (NPCs don't care about EVs), or Amulet Coin (NPCs don't care about money).</html>
+GUI.tpBossTrainersItemsCheckBox.toolTipText=<html>Check this to add or replace held items for the Pokemon of "Boss" Trainers.<br />Boss Trainers include Gym Leaders, Kahunas, Team Leaders, Elite Four, and Champions.
+GUI.tpImportantTrainersItemsCheckBox.toolTipText=<html>Check this to add or replace held items for the Pokemon of "Important" Trainers.<br />Important Trainers include Rivals/Friends, Team Admins, and other important story battles (such as Colress in BW2 and Sycamore in XY).
+GUI.tpRegularTrainersItemsCheckBox.toolTipText=<html>Check this to add or replace held items for the Pokemon of regular Trainers.<br />Regular Trainers include all trainers that don't qualify as "Boss" or "Important" Trainers.
+GUI.heldItemsConsumable.text=Consumable Only
+GUI.heldItemsConsumable.tooltip=<html>Items will be chosen randomly from all single-use held items that a Pokemon can eat/use on its own.<br />This is mostly berries, with a few other items like Focus Sash and White Herb.</html>
+GUI.heldItemsSensible.text=Sensible Items
+GUI.heldItemsSensible.tooltip=<html><b>Not implemented yet.</b><br />Items will be chosen randomly from a subset of the held items that "make sense" for a given Pokemon. For example, Silk Scarf won't be an option for a given Pokemon unless it knows at least one normal move. Another example is that a Yache berry will only be an option on Pokemon that are weak or double weak to ice.</html>
+GUI.heldItemsAllPokemon.text=All Pokemon
+GUI.heldItemsAllPokemon.tooltip=If this option is selected, a held item will be given to all of an applicable trainer's Pokemon.
+GUI.heldItemsHighestLevel.text=Highest Level Only
+GUI.heldItemsHighestLevel.tooltip=If this option is selected, a held item will be given to a trainer's highest level Pokemon (or one of them, if there are multiple at that level).
```

Comment:
Please make the variable names in the bundle consistent with how it works for all other GUI elements - `GUI.nameOfElement.text` and `GUI.nameOfElement.toolTipText`. (As in, the actual name of the element's variable in `NewRandomizerGUI`.)

Reply:
Done.